usaspending_api/references/models/dabs_submission_window_schedule.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+
+
+class DABSSubmissionWindowSchedule(models.Model):
+    id = models.AutoField(primary_key=True)
+    reporting_start = models.DateTimeField()
+    reporting_end = models.DateTimeField()
+    submission_start_date = models.DateTimeField()
+    submission_end_date = models.DateTimeField()
+    certification_end_date = models.DateTimeField()
```

Comment:
Do we want all of these to be datetimes or just dates?

Reply:
> Do we want all of these to be datetimes or would just dates suffice?

You know what... never mind.  I see they're timestamps in Broker.  Forget I asked.